prototype/genetic_curve_fitting_prototype/genetic_curve_fitting.py

```diff
@@ -0,0 +1,157 @@
+import time
+import struct
+import serial
+import numpy as np
+import csv
+import matplotlib.pyplot as plt
+
+
+"""Use calculated coefficiants to estimate the data point
+
+:param x The x data point to estimate ethe value
+:param parameters: The coefficients that are used (they are all between 0 and 1)
+:param limits: The maximum value of each coefficient in a list
+:param polynomialDegree: The maximum polynomial degree of the equation
+:returns: estimated y value
+"""
+def predict(x,parameters,limits, polynomialDegree):
```

Comment:
I will not comment every time it happens, but the standard indicates to put a space after a comma or between an operator (for example we would write `a = x + 2;` instead of `a=x+2). Did you ran clang-format ?